src/models/shopping-cart.model.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Entity, model, property} from '@loopback/repository';
+
+/**
+ * Item in a shopping cart
+ */
+export class ShoppingCartItem {
```

Comment:
I am afraid we don't generate OpenAPI schema for a class without decorating it with `@model` and `@property`.